wordpress-mobile/WordPress-Android
# [Blogging Prompts] Prompts List analytics tracking

Part of #17125

Add the `blogging_prompts_prompts_list_viewed` analytics event when the Prompts List Activity is opened, matching iOS event and tracking location.

The merging of this should happen after #17682 is merged.

To test:
1. Do the steps to enable the Prompts List feature flag found in the test section of https://github.com/wordpress-mobile/WordPress-Android/pull/17675
2. Sign in (if not signed in already)
3. Go to the home dashboard (My Site -> Home)
4. **Verify** the Prompts card is shown
5. Tap the 3-dot menu in the Prompts card
6. Tap the "View more prompts" option
7. **Verify** log for "Tracked: blogging_prompts_prompts_list_viewed"

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
Unit tests for new specific tracker.

PR submission checklist:
